.taplo.toml

```diff
@@ -0,0 +1,7 @@
+# `taplo` configuration
```

Comment:
i would rather keep the defaults of taplo

Reply:
It splits many of the dependency lines in the `Cargo.toml` files making it harder to keep them organized.

Reply:
Do you have examples of that?

For example, this one seems unrelated:
array_auto_collapse = false

Reply:
I added them iteratively as formatting problems cropped up. 
I don't remember the details for the need for that line.
It might have been needed before the final format was completed, and redundant now.
I'll remove that line it locally and see if anything crops back up.

Reply:
All of the configuration tags here are purposeful and help me keep the TOML organized and readable.